registry/accessing-the-registry.adoc

```diff
@@ -32,13 +32,15 @@ the user must have the `registry-viewer` role. To add this role:
 $ oc policy add-role-to-user registry-viewer <user_name>
 ----
 
-* For writing or pushing images, for example when using the `podman push` command,
-the user must have the `registry-editor` role. To add this role:
+* For writing or pushing images, for example when using the `podman push` command:
+** The user must have the `registry-editor` role. To add this role:
 +
 [source,terminal]
 ----
 $ oc policy add-role-to-user registry-editor <user_name>
 ----
++
+** Your cluster must have an existing project where the images can be pushed to.
```

Comment:
This could just be a note instead of its own bullet. 

```suggestion
+
[NOTE]
====
Your cluster must have an existing project where the images can be pushed to.
====
```

Reply:
As per my discussion with SME, this is an explicit prerequisite for performing a push. Hence, I avoided making this as note.

Reply:
> This could just be a note instead of its own bullet.

SME suggested that this should be an explicit prerequisite for pushing images hence avoided putting this up as a note.